packages/zosjobs/README.md

```diff
@@ -1,11 +1,48 @@
 # z/OS Jobs Package
+Contains APIs and commands to interact with jobs on z/OS (using z/OSMF jobs REST endpoints).
+
 
 Contains APIs to interact with jobs on z/OS (using z/OSMF jobs REST endpoints).
+const port: number = 443;
+const user: string = "ZOWEUSER";
+const password: string = "ZOWEPASS";
+const protocol: SessConstants.HTTP_PROTOCOL_CHOICES = "https";
+const basePath: string = undefined;
+const type: SessConstants.AUTH_TYPE_CHOICES = "basic";
+const tokenType: string = undefined;
+const tokenValue: string = undefined;
+const rejectUnauthorized: boolean = false;
```

Comment:
Looks like formatting in this file was messed up by merge, this (and other lines below) should be in a code block.